src/locales/cy/pages.yml

```diff
@@ -1,10 +1,10 @@
 intro:
   h1: Intro
 rti-payslip-national-insurance:
-  title: Enter the amount of National Insurance you paid from a recent payslip
-  inset: Use a payslip dated after {{ threeMonthsAgo }} to answer this question
+  title: Rhowch swm yr Yswiriant Gwladol a dalwyd gennych o slip cyflog diweddar
+  inset: Defnyddiwch slip cyflog wedi'i ddyddio ar ôl {{ threeMonthsAgo }} i ateb y cwestiwn hwn
   content:
-    - Tell us the amount of National Insurance you paid.
-    - You can usually find the amount next to any pensions or PAYE/income tax deductions on your payslip.
-    - Enter the amount in pounds and pence, exactly as it appears on your payslip.
-  abandon: "<a href='/kbv/abandon' class='govuk-link' data-id='abandon'>I need to prove my identity another way</a>"
+    - Dywedwch wrthym faint o Yswiriant Gwladol rydych wedi'i dalu.
+    - Fel arfer, gallwch ddod o hyd i'r swm wrth ymyl unrhyw bensiynau neu ddidyniadau TWE/Treth Incwm ar eich slip cyflog.
+    - Rhowch y swm mewn punnoedd a cheiniogau yn union fel y mae'n ymddangos ar eich slip cyflog.
+  abandon: "<a href='/kbv/abandon' class='govuk-link' data-id='abandon'>Mae angen i mi brofi fy hunaniaeth mewn ffordd arall</a>"
```

Comment:
`abandon` will be shared across several pages, could we pull it out of this page and put it under a `shared` key or something